doc/final_report.md

```diff
@@ -0,0 +1,90 @@
+
+```{r observations_group, echo=FALSE}
+knitr::kable(model_quality$table, caption = "Table 1. Number of observations in each group.")
+```
+
+
+To decide what kind of measure of central tendency to apply in our hypothesis test, we looked at the distribution of the sentence lengths of either group (COLORS) using a density plot t. As the distribution for both groups was right skewed, we decided to use the median as our parameter as it is less affected by extreme values as It based on the count.  
+
+```{r groups_sentence_distributions, echo=FALSE, fig.cap="Figure 1. The distribution of the aggregate sentence time of indigenous and non-indigenous inmates.”, out.width = '100%'}
+
+knitr::include_graphics("../results/“DISTRIBUTION.png")
+```
+
+Looking at the boxplot, we can see that there is a large overlap in the confidence intervals between the two groups, and potentially indicates that there is not a significant. 
+
+```{r groups_sentence_distributions, echo=FALSE, fig.cap="Figure 2. The distribution of the aggregate sentence time of indigenous and non-indigenous inmates.”, out.width = '100%'}
+
+knitr::include_graphics("../results/“DISTRIBUTION.png")
+```
+
+As we selected the median, we subsequently carried out a permutation test under the null hypothesis computationally with `r N_REPEATS`.  This model is applicable as we have a large sample size.
+
+The null hypothesis stated that there is no difference in the median sentence lengths between the indigenous and non-indigenous inmates. The alternate hypothesis stated that there is a difference in the median sentence lengths between the indigenous and non-indigenous populations. The test statistic was the difference in the sample medians of the two groups. The significance level was set to 0.05. 
+
+The distribution under the null hypothesis has the expected normal shape due to the sample size and number of repetitions.
+
+```{r groups_sentence_distributions, echo=FALSE, fig.cap="Figure 3. The null distribution generated using permutation of the difference in medians of the two groups.”, out.width = '100%'}
+
+knitr::include_graphics("../results/“NULL_DISTRIBUTION.png")
+```
+
+The difference in the sample test medians was `r DIFF_IN_MEANS`, which corresponded to a p-value of  `r P_VALUE`. As the p-value was greater than the significance level of 0.05, we do not have enough evidence to reject the null hypothesis. Hence there is no statistically significant difference in the median sentence lengths between indigenous and non-indigenous groups. 
```

Comment:
The difference in the sample test medians was `r DIFF_IN_MEANS`, which corresponded to a p-value of  `r P_VALUE`. As the p-value was greater than the significance level of 0.05, we do not have enough evidence to reject the null hypothesis. Hence, we conclude that there is no statistically significant difference in the median sentence lengths between indigenous and non-indigenous groups.